templates/default/apps-version6.conf.erb

```diff
@@ -0,0 +1,24 @@
+# This file was generated by Chef
+
+$ModLoad imfile
+$InputFilePollInterval <%= node.loggly.rsyslog.input_file_poll_interval %>
+$WorkDirectory <%= node.rsyslog.working_dir %>
+
+# Add a tag for <%= @app_name %> app events
+$template <%= @format %>,"<%%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [<%= node.loggly.token %>@41058 tag=\"<%= @app_name %>\"] %msg%\n"
+
+<%- @log_files.each do |logfile| -%>
+# Input for <%= logfile['filename'] %>
+$InputFileName <%= logfile['filename'] %>
+$InputFileTag <%= logfile['tag'] %>
+$InputFileStateFile <%= logfile['statefile'] %>
+$InputFilePersistStateInterval 20000
+$InputRunFileMonitor
+
+<%- end -%>
+
+# Send to Loggly then discard
+<%- @file_tags.each do |file_tag| -%>
+if $programname == '<%= file_tag %>' then @@<%= node.loggly.rsyslog.host %>:<%= node.loggly.rsyslog.port %>;<%= @format %>
+if $programname == '<%= file_tag %>' then ~
+<%- end -%>
```

Comment:
You can keep them separate if you'd like due to the `apps` convention. I'm not sure if it would really be beneficial to abstract into another template or keep the duplication for the sake of being explicit. What do you think?